static/compiler.js

```diff
@@ -72,7 +72,7 @@ define(function (require) {
         this.domRoot.html($('#compiler').html());
         this.id = state.id || hub.nextCompilerId();
         this.sourceEditorId = state.source || 1;
-        this.currentLangId = state.lang || "c++";
+        this.currentLangId = state.lang || "php";
```

Comment:
This shouldn't be hard coded

Reply:
Resolved some time ago in a newer unification commit (f08ceb9 to be exact)